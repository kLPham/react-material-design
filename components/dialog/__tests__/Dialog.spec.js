import React from 'react';
import { shallow } from 'enzyme';
import Dialog from '../index';

describe('<Dialog />', () => {
    it('should render', () => {
        const dialog = shallow(
            <Dialog />,
    );
        expect(dialog).toMatchSnapshot();
    });
});
