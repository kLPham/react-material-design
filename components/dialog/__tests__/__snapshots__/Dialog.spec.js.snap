// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Dialog /> should render 1`] = `
<aside
  aria-describedby="my-mdc-dialog-description"
  aria-labelledby="my-mdc-dialog-label"
  className="mdc-dialog"
  id="my-mdc-dialog"
  role="alertdialog"
  style={
    Object {
      "visibility": "hidden",
    }
  }
>
  <div
    className="mdc-dialog__surface"
  >
    <header
      className="mdc-dialog__header"
    >
      <h2
        className="mdc-dialog__header__title"
        id="my-mdc-dialog-label"
      >
        Use Googles location service?
      </h2>
    </header>
    <section
      className="mdc-dialog__body mdc-dialog__body--scrollable"
      id="my-mdc-dialog-description"
    >
      Let Google help apps determine location. This means sending anonymous location data to Google, even when no apps are running.
    </section>
    <footer
      className="mdc-dialog__footer"
    >
      <button
        className="mdc-button mdc-dialog__footer__button mdc-dialog__footer__button--cancel"
        type="button"
      >
        Decline
      </button>
      <button
        className="mdc-button mdc-dialog__footer__button mdc-dialog__footer__button--accept"
        type="button"
      >
        Accept
      </button>
    </footer>
  </div>
  <div
    className="mdc-dialog__backdrop"
  />
</aside>
`;
